### PR TITLE
Shared classes

### PR DESCRIPTION
**Pull request by karimosman**

### CODE REVIEW

**Comment by karimosman** [comment]
@svenvanhal Ready to merge

File: Timetabling.Tests/Objects/ActiviteitsListTest.cs [hunk 1]
```diff
@@ -54,14 +56,40 @@ public void Init()
...
             mockDB.Setup(item => item.School_Lookup_Class).Returns(mockSet2.Object);
             mockDB.Setup(item => item.Subject_SubjectGrade).Returns(mockSet3.Object);
+            mockDB.Setup(item => item.TeacherClassSubjectGroups).Returns(mockSet4.Object);
+            mockDB.Setup(item => item.Tt_ClassGroup).Returns(mockSet5.Object);
+
             var list = new ActivitiesList(mockDB.Object);
             list.Create();
             test = list.GetList();
-
+            System.Console.WriteLine(test);
```
Comment (svenvanhal) [inline Timetabling.Tests/Objects/ActiviteitsListTest.cs]: remove

**Comment by karimosman** [comment]
@svenvanhal ready to merge

**Comment by karimosman** [comment]
dont merge this one

**Pull request closed without merging**